MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -199,6 +199,23 @@ The `Signature` property is of type [InkAnnotation](https://help.syncfusion.com/
 
 Button form fields will be rendered in the PDF viewer. But the PDF viewer supports only the `GoTo` actions that navigates to a particular location in the PDF document alone. Other types of button actions are not supported.
 
+### Adding custom information to a form field
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to assign custom information to form fields. The `CustomData` property is utilized to store additional information about each form field instance for reference. However, it's essential to note that these data are solely intended for reference purposes and will not be displayed in the `SfPdfViewer` interface.
+
+The following code sample demonstrates how to set the `CustomData` property for a form field. In this code snippet, we set the custom data to represent the modification time of the form field.
```

Comment:
The following code sample demonstrates how to set the `CustomData` property for a form field. In this code sample, we set the custom data to represent the modification time of the form field.